winter/__init__.py

```diff
@@ -27,14 +27,12 @@ def _default_configuration():
     from .schema import PathParametersInspector
     from .schema import QueryParametersInspector
     from .schema import register_controller_method_inspector
-    from . import schema
 
     arguments_resolver.add_argument_resolver(DRFBodyArgumentResolver())
     arguments_resolver.add_argument_resolver(QueryParameterResolver())
     arguments_resolver.add_argument_resolver(HttpRequestArgumentResolver())
     register_controller_method_inspector(PathParametersInspector())
     register_controller_method_inspector(QueryParametersInspector())
```

Comment:
There are two verbs here:
* `resolve` which is used to find out a value for a particular parameter based on some external data.
* `inspect` which is used to decide type schema based on a type itself.

If it works for you, then please rename `type_hint.py` module members to something with `inspect` word usage and avoid using `resolve` to avoid naming and meaning collision.

For example:
`schema/type_hint.py` -> `schema/type_inspection.py`
`schema/test_argument_type_info.py` -> `schema/test_type_inspection.py`
`float_resolver` -> `inspect_float`
`get_basic_type_info_from_hint` -> `inspect_type`